src/lib/Sendbird.tsx

```diff
@@ -91,13 +91,13 @@ export interface SendbirdProviderProps extends CommonUIKitConfigProps {
 
 function Sendbird(props: SendbirdProviderProps) {
   const {
-    replyType = 'NONE',
-    isMentionEnabled = false,
-    isReactionEnabled = true,
-    disableUserProfile = false,
-    isVoiceMessageEnabled = true,
-    isTypingIndicatorEnabledOnChannelList = false,
-    isMessageReceiptStatusEnabledOnChannelList = false,
```

Comment:
Don't do this anymore in here since this(it's P3) will break the config priority rule.

```
- 
    - **[P3]** UIKit default value
    - **[P2]** Values set from Dashboard
    - **[P1]** Values set in code by the customer
```

By removing this P3 default value setting logic, we make P2 config value beat P3 via UIKitConfigProvider.